convert_lora_to_gguf.py

```diff
@@ -257,11 +257,11 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument(
         "--base", type=Path, required=True,
-        help="directory containing base model file",
+        help="directory containing Hugging Face model config files (config.json, tokenizer.json) for the base model that the adapter is based on - only config is needed, actual model weights are not required",
     )
     parser.add_argument(
         "lora_path", type=Path,
-        help="directory containing LoRA adapter file",
+        help="directory containing Hugging Face PEFT LoRA config (adapter_model.config) and weights (adapter_model.safetensors or adapter_model.bin)",
```

Comment:
```suggestion
        help="directory containing Hugging Face PEFT LoRA config (adapter_config.json) and weights (adapter_model.safetensors or adapter_model.bin)",
```

Example of a PEFT adapter: https://huggingface.co/mlabonne/Llama-3-70B-Instruct-abliterated-LORA/tree/main

Reply:
Sorry, that was a silly mistake - fixed now.